engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoObject.java

```diff
@@ -22,6 +22,6 @@ public Class<? extends TruffleLanguage<?>> getLanguage() {
 
   @ExportMessage
   public Object toDisplayString(boolean allowSideEffects) {
-    throw new UnsupportedOperationException("unimplemented");
+    return toString();
```

Comment:
You cannot all `Object.toString()` in PE mode.

Reply:
Fixed by 7fa2022b8c6fd657057bc8da504e5c919df843ef